packages/core/src/exposed-thing.ts

```diff
@@ -67,9 +67,48 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
         // without functions or methods
         let clonedModel = JSON.parse(JSON.stringify(thingModel))
         Object.assign(this, clonedModel);
+
+        // unset "@type":"tm:ThingModel" (see https://github.com/eclipse/thingweb.node-wot/issues/426)
+        if (this["@type"]) {
+            if (typeof this["@type"] === 'string' && this["@type"] === "tm:ThingModel") {
+                delete this["@type"];
+            } else if (Array.isArray(this["@type"])) {
+                let arr: Array<any> = this["@type"];
+                for (var i = 0; i < arr.length; i++) {
+                    if (arr[i] === "tm:ThingModel") {
+                        arr.splice(i, 1);
+                        i--;
+                    }
+                }
+            }
+        }
+        // set default language
+        this.addDefaultLanguage(this);
+        // extend interactions
```

Comment:
I would approve the change but see my comment: https://github.com/eclipse/thingweb.node-wot/issues/426#issuecomment-859660742